benefits/templates/admin/login.html

```diff
@@ -0,0 +1,28 @@
+{% extends "admin/login.html" %}
+{% load i18n static %}
+
+{% block dark-mode-vars %}
+{% endblock dark-mode-vars %}
+
+{% block extrastyle %}
+    <link href="{% static "img/favicon.ico" %}" rel="icon" type="image/x-icon" />
+    <script nonce="{{ request.csp_nonce }}"
+            src="https://cdn.jsdelivr.net/npm/jquery@3.6.1/dist/jquery.min.js"
+            integrity="sha256-o88AwQnZB+VDvE9tvIXrMQaPlFFSUTR+nldQm1LuPXQ="
+            crossorigin="anonymous"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+          rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+          crossorigin="anonymous">
+    <link href="{% static "css/admin/styles.css" %}" rel="stylesheet">
+{% endblock extrastyle %}
```

Comment:
Question! @lalver1 @angela-tran - I put this here b/c the login.html template is not already inheriting the extrastyle block from agency-base. Why is that? Is a way I can get login.html to inherit from agency-base?

Also note here: I had to move the call to `styles.css` to line 17 so that the variable overrides set in there actually do override the Bootstrap variables. That's why I also made this same change in agency-base.

Reply:
@machikoyasuda I think the reason is that `benefits/templates/admin/login.html` extends `contrib/admin/login.html` and `agency-base` is not in that 'chain' of inheritance (`agency-base` extends `contrib/admin/base.html`) so `login.html` is not inheriting the extrastyle block from `agency-base`.

What you have works, but I guess you could also have `benefits/templates/admin/login.html` extend `agency-base` and duplicate some code from `contrib/admin/login.html` into `benefits/templates/admin/login.html`. Maybe that duplication would be ok since we are basically overriding the whole `login` template.

Reply:
I tested this alternate approach about extending `agency-base.html` instead and `login.html` would look like this (we no longer need the `extrastyle` block):
```
{% extends "admin/agency-base.html" %}
{% load i18n static %}

{% block bodyclass %}
    {{ block.super }} login
{% endblock bodyclass %}

{% block usertools %}
{% endblock usertools %}

{% block nav-global %}
{% endblock nav-global %}

{% block content %}
    {% if form.errors and not form.non_field_errors %}
        <p class="errornote">
            {% blocktranslate count counter=form.errors.items|length %}Please correct the error below.{% plural %}Please correct the errors below.{% endblocktranslate %}
        </p>
    {% endif %}

    {% if form.non_field_errors %}
        {% for error in form.non_field_errors %}<p class="errornote">{{ error }}</p>{% endfor %}
    {% endif %}

    <div id="content-main">

        {% if user.is_authenticated %}
            <p class="errornote">
                {% blocktranslate trimmed %}
                    You are authenticated as {{ username }}, but are not authorized to
                    access this page. Would you like to login to a different account?
                {% endblocktranslate %}
            </p>
        {% endif %}

        <form action="{{ app_path }}" method="post" id="login-form">
            {% csrf_token %}
            <div class="form-row">
                {{ form.username.errors }}
                {{ form.username.label_tag }} {{ form.username }}
            </div>
            <div class="form-row">
                {{ form.password.errors }}
                {{ form.password.label_tag }} {{ form.password }}
                <input type="hidden" name="next" value="{{ next }}">
            </div>
            {% url 'admin_password_reset' as password_reset_url %}
            {% if password_reset_url %}
                <div class="password-reset-link">
                    <a href="{{ password_reset_url }}">{% translate "Forgotten your password or username?" %}</a>
                </div>
            {% endif %}
            <div class="submit-row">
                <input type="submit" value="{% translate "Log in" %}">
            </div>
        </form>

    </div>
{% endblock content %}

{% block branding %}
    <div class="d-flex justify-content-center w-100 bg-primary">
        <img class="my-3" src="{% static "img/logo-lg.svg" %}" width="220" height="50" alt="California Integrated Travel Project: Benefits logo (large)" />
    </div>

    <div id="site-name">
        <h1 class="text-center text-white fs-3 py-3 m-0">{{ site_header }}</h1>
    </div>
{% endblock branding %}
```

Reply:
Thank you for checking this! It makes sense to me now, how the inheriting happens. I think I want to err on the side of caution of not wanting to accidentally import jQuery twice, so I decided I actually don't want to inherit `extrastyles` from `agency-base`. So I am good with how the `login.html` template looks like now: minimal block overriding, just overriding `branding` basically, and an `extrastyles` that removes the jQuery script.